src/main/java/rx/Observable.java

```diff
@@ -1453,6 +1453,270 @@ public Completable toCompletable() {
     }
 
     /**
+     * Concatenates two Observables into a single sequence by subscribing to each Observable,
```

Comment:
Could you update the wording according to the [plain `concat`](https://github.com/ReactiveX/RxJava/blob/1.x/src/main/java/rx/Observable.java#L1132) methods as there are no outer and inner Observables with these methods?